packages/bitcore-payid/src/index.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  AddressDetailsType,
+  verifySignedAddress,
+} from '@payid-org/utils';
+import { JWS } from 'jose';
+import { IVerifyPayId } from './index.d';
+
+class PayId {
+  constructor() {}
+
+  /**
+   * Sign a payId address with the given identityKey
+   * @param {string} payId
+   * @param {string} address Address to be signed
+   * @param {string} currency Currency ticker (e.g. BTC, ETH, XRP)
+   * @param {string} identityKey The key to be used for signing
```

Comment:
maybe specify what type of key is expected? (ie:  ecdsa 32byte priv key) and an example of input

Reply:
Done, though let me know if you don't like what I added. PayId tries to be key agnostic, so it should be able to accept any key string.